pkg/certificate/types.go

```diff
@@ -53,24 +55,25 @@ type Certificate struct {
 	IssuingCA pem.RootCertificate
 }
 
-// Manager is the interface declaring the methods for the Certificate Manager.
-type Manager interface {
-	// IssueCertificate issues a new certificate.
-	IssueCertificate(CommonName, time.Duration) (*Certificate, error)
-
-	// GetCertificate returns a certificate given its Common Name (CN)
-	GetCertificate(CommonName) (*Certificate, error)
-
-	// RotateCertificate rotates an existing certificate.
-	RotateCertificate(CommonName) (*Certificate, error)
-
-	// GetRootCertificate returns the root certificate in PEM format and its expiration.
-	GetRootCertificate() (*Certificate, error)
+type Manager struct {
```

Comment:
let's add a new file, certificate/manager.go

Here we will add all the methods onto the Manager struct (every method of the old Manager interface, will be a method on the new Manager struct).

These methods should look very similar to the old methods